src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        StringBuilder stringBuilderReader = new StringBuilder();
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        String report = null;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                stringBuilderReader.append(value).append(",");
+                value = reader.readLine();
+            }
+            String [] fromStringBuilderReader = stringBuilderReader.toString().split(",");
+            for (int i = 0; i < fromStringBuilderReader.length; i++) {
+                if (fromStringBuilderReader[i].equals("supply")) {
+                    supply += Integer.parseInt(fromStringBuilderReader[i + 1]);
```

Comment:
You can increase index immideately (++i) to decrease loop step. Now your loop should get through this if-statements every time, even if it number.

Reply:
@nacenik did you mean "supply" / "buy" to replace with the constant?

Reply:
Yes